deploy/resources.go

```diff
@@ -0,0 +1,8 @@
+package resources
```

Comment:
This is not related to this PR, is it?

Reply:
right, my bad, I created this branch on top of an existing one, for something not completed yet 🤦‍♂️

Reply:
removed in 28ec1eb